config.json

```diff
@@ -260,8 +260,8 @@
       {
         "groupId": "androidx.core",
         "artifactId": "core-role",
-        "version": "1.0.0-rc01",
-        "nugetVersion": "1.0.0.4-rc01",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0",
```

Comment:
Just a heads up that this `nugetVersion` will need to be bumped to `1.0.0.4+` to replace the previous version.